### DISCUSSION TITLE
Grain boundary plane distribution

### DISCUSSION THREAD

**Discussion by mugikumar** [mtex-toolbox/mtex#1788] [Ask Anything]
Hi developers
               I am trying to do the five-grain boundary parameter in Mtex, but I need help. If you have any ideas, please suggest to me to overcome this. I have attached the image. Please respond to me as soon as possible
![mtex discussion](https://user-images.githubusercontent.com/109027461/230718271-ad1b0507-67cd-41e1-bf59-59c50cffeb7e.jpg)

**Comment by ralfHielscher** [comment]
What are your input data?

**Comment by mugikumar** [comment]
Hi ralf, 
          I have to input the data of different miller indices and plot the pole density function with different intervariant misorientations degrees. Can you please provide the code for the white lines?

**Comment by ralfHielscher** [comment]
For the five-parameter grain boundary distribution you require the a list of boundary normals as well as the corresponding orientations. Just some Miller indices are not sufficient.

Ralf.

**Comment by mugikumar** [comment]
Hi ralf
I am sorry for the late reply. I am in particular busy with my personal problem
Five parameter grain boundary distribution- three of them are lattice misorientation, and two of them are boundary plane orientation. For lattice misorientation can quickly get from conventional EBSD, boundary plane orientation needs 3D EBSD. G.S Rohrer et al. suggested with the help of a stereology approach analysing the boundary plane distribution means found either tilt or twist boundaries. In mtex documentation also mentioned is possible to get the trace for tilt boundaries.  I need a code for the pole figure with grain boundary distribution for different plane normal means I am analyzing how changes crystallographic plane for intervariant boundaries.  the input data are variant boundaries are misoriented by 60 degree and rotated by direction of [11-1] Euler angle must be defined by grain boundaries what are misoriented by 60 degree  
![mtex doubt](https://user-images.githubusercontent.com/109027461/233602537-636ca48f-2cee-4d0f-a65c-e98fd8c99228.jpg)
